politeiawww/memorycache.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"sync"
+	"time"
+)
+
+// Item is a cached reference
+type Item struct {
+	Content    []byte
+	Expiration int64
+}
+
+//Storage mechanism for caching strings in memory
+type Storage struct {
+	items map[string]Item
+	mu    *sync.RWMutex
```

Comment:
Best practices for this is to define the mutex directly on the struct and to use pointer receivers instead of value receivers like you've got.  Ex:

```
type Storage struct {
	sync.RWMutex
	items map[string]Item
}


func (s *Storage) Get(key string) []byte {
	s.RLock()
	defer s.RUnlock()
	...
}
```

Reply:
not sure if I got this right. Please check.